ros2/Dockerfile

```diff
@@ -85,21 +86,19 @@ ENV TINI_VERSION v0.9.0
 ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /bin/tini
 RUN chmod +x /bin/tini
 
-RUN apt-get update \
-  && apt install -y libgazebo9-dev ros-${ROS_DISTRO}-gazebo-ros-pkgs \
-  && apt-get clean \
-  && rm -rf /var/lib/apt/lists/*
-
 RUN apt-get update \
   && apt install -y \
     libgazebo9-dev \
     ros-${ROS_DISTRO}-gazebo-ros-pkgs \
+    rti-connext-dds-${RTI_VERSION} \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
+# Make the default workspace somewhere other than /root, and do not copy the examples
+RUN mv /rticommon_config.sh /opt/rti.com/rti_connext_dds-${RTI_VERSION}/resource/scripts/rticommon_config.sh
 
 # build package
 ARG DIRECTORY
-COPY "${DIRECTORY}" /.dockerinstall
+COPY "${DIRECTORY}"/pkgs.rosinstall "${DIRECTORY}"/apt.list* /.dockerinstall/
```

Comment:
"${DIRECTORY}/pkgs.rosinstall"
"${DIRECTORY}/apt.list"

Reply:
We probably want to fix these quotes and add a comment above this line that explains the clever hack that's going on.

Reply:
Done